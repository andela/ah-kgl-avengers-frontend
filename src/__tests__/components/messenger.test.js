import React from 'react';
import { shallow, mount } from 'enzyme';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';
import thunk from 'redux-thunk';
import Messenger from '../../components/editor/message';

const mockStore = configureMockStore([thunk]);
const store = mockStore({});

const props = {
  welcome: jest.fn(),
  response: jest.fn(),
};
describe('<Chips />', () => {
  beforeEach(() => {});

  afterEach(() => {
    jest.clearAllMocks();
  });
  const component = shallow(<Messenger messages={['node', 'es6']} type="error" />);
  it('should render without the component', () => {
    expect(component).toMatchSnapshot();
  });

  test('should mount the editor', () => {
    const wrapper = mount(<Messenger messages={['node', 'es6']} type="error" />);
    expect(wrapper.find('Messenger')).toBeDefined();
  });
});
