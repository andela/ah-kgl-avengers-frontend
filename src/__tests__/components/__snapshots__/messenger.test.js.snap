// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Chips /> should render without the component 1`] = `ShallowWrapper {}`;
